MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -0,0 +1,275 @@
+import Polyline
+import MapboxDirections
+import AVFoundation
+import MapboxMobileEvents
+
+let SecondsBeforeCollectionAfterFeedbackEvent: TimeInterval = 20
+
+struct EventDetails {
+    var originalRequestIdentifier: String?
+    var requestIdentifier: String?
+    var coordinate: CLLocationCoordinate2D?
+    var originalGeometry: Polyline?
+    var originalDistance: CLLocationDistance?
+    var originalEstimatedDuration: TimeInterval?
+    var originalStepCount: Int?
+    var geometry: Polyline?
+    var distance: CLLocationDistance?
+    var estimatedDuration: TimeInterval?
+    var stepCount: Int?
+    var created: Date
+    var startTimestamp: String?
```

Comment:
This should be a Date, like `created`. Wait until `convertedToDictionary()` to lazily convert it to a string.